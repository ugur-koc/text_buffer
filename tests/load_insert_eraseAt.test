load aaa.txt
insert 2 do 
eraseAt 2 3
exit