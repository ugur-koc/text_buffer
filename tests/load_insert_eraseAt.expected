Available Operations: insert append eraseAt erase replace load save help exit 
Text Buffer: 
Text Buffer: I love you
Text Buffer: I do love you
Text Buffer: I love you
